### PR TITLE
ImageMagick: update to 7.1.0.43.

### PR DESCRIPTION
**Pull request by MouadCharradi**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86-64, glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64

### CODE REVIEW

File: srcpkgs/ImageMagick/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'ImageMagick'
 pkgname=ImageMagick
-version=7.1.0.41
+version=7.1.0.43
 revision=1
 _majorver=${version%.*}
 _patchver=${version##*.}
-wrksrc="${pkgname}-${_majorver}-${_patchver}"
+wrksrc="${pkgname}-${_majorver}.${_patchver}"
```
Comment (classabbyamp) [inline srcpkgs/ImageMagick/template:7]: I think this is the default wrksrc value now, can you remove this line to check that?

File: srcpkgs/ImageMagick/template [hunk 2]
```diff
@@ -19,8 +19,8 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="ImageMagick"
 homepage="https://www.imagemagick.org"
 changelog="https://imagemagick.org/script/changelog.php"
-distfiles="https://github.com/ImageMagick/ImageMagick/archive/${_majorver}-${_patchver}.tar.gz"
-checksum=f8ec06b5c82cbab4734b166b820f151323c78f438580daa176cedc63990c00a2
+distfiles="https://github.com/ImageMagick/ImageMagick/archive/${_majorver}.${_patchver}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/ImageMagick/template:22]: same thing here, you should just be able to use $version

File: srcpkgs/ImageMagick/template (hunk 1 above)
Comment (MouadCharradi) [reply srcpkgs/ImageMagick/template:7]: You mean I should get rid off the _patchver and _majorver variables and just use version as is?

Comment (classabbyamp) [reply srcpkgs/ImageMagick/template:7]: yes, unless it's used later in the template

**Comment by MouadCharradi** [comment]
Is this how it should be now?

**Comment by classabbyamp** [comment]
yeah looks good

**Pull request merged**